### DISCUSSION TITLE
sync watched for missing media

### DISCUSSION THREAD

**Discussion by filviu** [Taxel/PlexTraktSync#1876] [Q&A] [answered]
Is it possible to sync watched status for removed media.

E.g.:

I have a long running tv show with 20 seasons. I only have files for the last season. If I go to the show in discover I can see plex knowing that all 20 seasons are watched - yet only the existing files get the watched status synced.

**Comment by glensc** [comment, marked as answer]
if you only deleted the files, i.e. plex episode is still in plex (with file found status), then it behaves the same. but if plex episode is also removed, then that is not supported.

for that, there's already a feature request:
- https://github.com/Taxel/PlexTraktSync/issues/1142

it has very same title what you're asking, did you even search open issues and discussions?

**Comment by filviu** [comment]
Thank you for the answer. 

Yes I did search but didn't find it.
